helmfile/charts/log-manager/templates/cronjob-compaction.yaml

```diff
@@ -28,6 +28,7 @@ spec:
             - name: compaction
               imagePullPolicy: {{ $.Values.image.pullPolicy }}
               image: {{ $.Values.image.repository }}:{{ default $.Chart.AppVersion $.Values.image.tag }}
+              workingDir: /scratch
```

Comment:
This might need to be conditional as well, or does it work properly without the ephemeral volume enabled?